src/com/xilinx/rapidwright/design/tools/RelocationTools.java

```diff
@@ -0,0 +1,272 @@
+package com.xilinx.rapidwright.design.tools;
+
+import java.util.*;
```

Comment:
Let's replace `import *`-style with explicit types.

Reply:
Ooh yes, sorry about that -- you've told me off before! I've changed my IDE to stop doing this now.